src/main/java/com/hbm/inventory/recipes/BlastFurnaceRecipes.java

```diff
@@ -63,7 +63,13 @@ public void registerDefaults() {
 		addRecipe(CO,									ModBlocks.block_meteor,							new ItemStack(ModItems.ingot_meteorite));
 		addRecipe(ModItems.meteorite_sword_hardened,	CO,												new ItemStack(ModItems.meteorite_sword_alloyed));
 		addRecipe(ModBlocks.block_meteor,				CO,												new ItemStack(ModItems.ingot_meteorite));
-		
+
+		addRecipe(ModItems.cobalt_sword, ModItems.ingot_meteorite, new ItemStack(ModItems.cobalt_decorated_sword));
```

Comment:
Better idea: Make an anvil smithing recipe with heated meteorite instead of shoving tools with wood handles into a miniature crucible

Reply:
Sure, forgot about the wood par

Reply:
The arirals are coming for you

Reply:
> Better idea: Make an anvil smithing recipe with heated meteorite instead of shoving tools with wood handles into a miniature crucible

![image](https://github.com/HbmMods/Hbm-s-Nuclear-Tech-GIT/assets/105080577/92da8395-4e08-476d-9d59-20488645bbc5)
there we go, thanks for reminding me about combustion